Cargo.toml

```diff
@@ -479,7 +479,7 @@ alloy-transport-ws = { version = "0.8.1", default-features = false }
 # op
 op-alloy-rpc-types = "0.8.3"
 op-alloy-rpc-types-engine = "0.8.3"
-op-alloy-rpc-jsonrpsee = "0.8.3"
+op-alloy-rpc-jsonrpsee = { git = "https://github.com/ffddw/op-alloy.git", branch = "issues/fix-return-type" } # TODO: to be replaced
```

Comment:
I release a new version

Reply:
Thank you! would it works with https://github.com/paradigmxyz/reth/pull/13423/commits/4a648d117d0ec399407edb6b9d37d2541c46f77c?